src/scripts/cancel.sh

```diff
@@ -1,5 +1,23 @@
 #!/bin/bash
 
+# Update and install packages using apk on Alpine Linux
+if command -v apk &> /dev/null; then
+  sudo apk update && sudo apk add jq curl
+
+# Update and install packages using yum on CentOS
+elif command -v yum &> /dev/null; then
+  sudo yum update && sudo yum install jq curl
+
+# Update and install packages using apt on Debian-based systems
+elif command -v apt-get &> /dev/null; then
+  sudo apt update && sudo apt install jq curl
+
+# If none of the package managers are found
+else
+  echo "Unsupported package manager"
+  exit
+fi
```

Comment:
```suggestion
else
  echo "Unsupported package manager. Please install jq and curl before run this orb."
fi
```
I think we don't need to stop in this case, just have a note so user know that they will need to install jq and curl by them self